### PR TITLE
EDGE-630 Send the voiceCallId to the generateTransferBxml method to h…

### PR DESCRIPTION
**Pull request by acopeland-bw**
…elp with debugging

### CODE REVIEW

**Comment by carrington-bw** [comment]
This looks good but we shouldn't merge until we merge the node-webrtc changes, cut a release, and update package.json

**Review by psherk-bw — APPROVED** [review]

**Review by carrington-bw — APPROVED** [review]

**Pull request merged**
